components/user_settings/display/user_settings_display.tsx

```diff
@@ -167,6 +169,17 @@ export default class UserSettingsDisplay extends React.PureComponent<Props, Stat
         }
     }
 
+    trackChangeIfNecessary(preference: PreferenceType, oldValue: any): void {
+        const props = {
+            field: 'display.' + preference.name,
```

Comment:
`PreferenceType.category` could possibly be used here (potentially making it reusable, and could be hoisted to something like `components/user_settings/utils`?)

Reply:
@calebroseland I agree upon using constants, but the naming is particular to how we use tracking already.
So `display.collapsed_reply_threads` conforms to our tracking naming in other places as well.
Gathering all these strings into a constants file though is making a lot of sense, 
I'll add another ticket to refactor track events to use constants.